src/layouts/App.vue

```diff
@@ -113,7 +113,7 @@
         </div>
       </div>
     </div>
-    <div class="flex flex-1 flex-col md:pl-64 min-h-[100vh] relative">
+    <div class="flex flex-1 flex-col md:pl-64 min-h-[100vh]">
```

Comment:
@darmhoo did you check this on mobile before removing the class relative? Hope it doesn't have any side effects.